rebar.config

```diff
@@ -1,21 +1,21 @@
 {erl_opts, [debug_info]}.
 {deps, [
-	{capwap, ".*", {git, "git://github.com/travelping/capwap", {tag, "1.9.1"}}},
-	{gen_listener_tcp, ".*", {git, "git://github.com/travelping/gen_listener_tcp", "master"}},
-	{netdata, ".*", {git, "git://github.com/RoadRunnr/erl_netdata", "master"}}
+    {capwap,           {git, "git://github.com/travelping/capwap",
+                           {branch, "master"}}},
+    {gen_listener_tcp, {git, "git://github.com/travelping/gen_listener_tcp",
+                           {branch, "master"}}},
+    {netdata,          {git, "git://github.com/RoadRunnr/erl_netdata",
+                           {branch, "master"}}}
```

Comment:
updateing the deps style is ok, but avaid the line break and don't tabulate the entries

Reply:
adjusted